doc/specs/vulkan/chapters/memory.txt

```diff
@@ -1721,6 +1721,7 @@ ifdef::VK_KHX_device_group[]
   * [[VUID-vkMapMemory-memory-00683]]
     pname:memory must: not have been allocated with multiple instances.
 endif::VK_KHX_device_group[]
+  * pname:ppData must: be a valid pointer to a pointer
```

Comment:
> There's a hack in the script. It reuses the "pointer to " string and erases the last 4 char.

Oh yea I remember that. Knew it would bite me in the ass one day D:

> I could make the script do it implicitly, but considering the weirdness of void and that it is a one occurence of void* without len it is perhaps better of as explicit validity.

Yea that's fair.

> Perhaps the vk.xml parsing script should throw error/exception when there is a void* without len && without noautovalidity.

Hmm. Let me have a think on that one, you're probably right though.